core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -91,34 +91,34 @@
     {% if username %}
       <li id="tutorialPreviewTab" ng-class="{'active': getTabStatuses().active === 'preview'}">
         <a href="#" tooltip="Preview" tooltip-placement="bottom" ng-click="selectPreviewTab()" class="protractor-test-preview-tab">
-          <span class="glyphicon glyphicon-play"></span>
+          <i class="material-icons">&#xE037;</i>
```

Comment:
Whoops, sorry -- I think Github was showing me only a partial list of changes, and I didn't see your changes to oppia.css. They look good! Sorry about that.